### PR TITLE
feature/#256: Added tests for CustomerOrdersViews

### LINKED ISSUE #256: SP: 1 Create tests for a view for retrieving all orders of the Customer

### PR DESCRIPTION
**Pull request by Misha86**
Added tests for CustomerOrdersViews.

### CODE REVIEW

File: beauty/api/tests/test_order_views.py [hunk 1]
```diff
@@ -219,3 +219,63 @@ def test_get_method_retrieve_order_logged_specialist(self):
...
+    def test_get_method_get_customer_orders_not_logged_user(self):
+        """Not logged users can not rearview a customer's orders."""
+        self.client.force_authenticate(user=None)
+        response = self.client.get(
+            path=reverse("api:customer-orders-list", args=[self.customer.id]),
+        )
+        self.assertEqual(response.status_code, 401)
+
+    def test_get_method_get_customer_orders_logged_not_customer(self):
+        """Logged users can not rearview a customer's orders."""
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_order_views.py]: can't view

**Review by pazuzu-ua — APPROVED** [review]

**Review by morento101 — APPROVED** [review]

**Comment by morento101** [comment]
You forgot to list new tests in module docstr

File: beauty/api/tests/test_order_views.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 from djoser.utils import encode_uid
```
Comment (shv833) [inline beauty/api/tests/test_order_views.py:43]: as it was said by morento101, You forgot to list new tests in module docstr

everything else looks good😎

**Review by shv833 — CHANGES_REQUESTED** [review]

File: beauty/api/tests/test_order_views.py (hunk 2 above)
Comment (Misha86) [reply beauty/api/tests/test_order_views.py:43]: Done

**Review by shv833 — APPROVED** [review]

**Pull request merged**